.github/workflows/custom_docker_builds.yml

```diff
@@ -44,23 +44,27 @@ jobs:
             image-tags: ghcr.io/spack/cache-indexer:0.0.1
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@96f53100ba2a5449eb71d2e6604bbcd94b9449b5 # v3.5.3
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@2b82ce82d56a2a04d2637cd93a637ae1b359c0a7 # v2.2.0
 
       - name: Log in to the Container registry
-        uses: docker/login-action@d398f07826957cd0a18ea1b059cf1207835e60bc
+        uses: docker/login-action@a9794064588be971151ec5e7144cb535bcb56e36 # v2.2.0
```

Comment:
To me, this looks like a [commit](https://github.com/docker/login-action/commit/a9794064588be971151ec5e7144cb535bcb56e36) on `master`, but maybe I'm not reading it right?  For the tagged release version `2.2.0` (I'm looking [here](https://github.com/docker/login-action/releases/tag/v2.2.0)), I would have thought you'd want `465a07811f14bebb1938fbed4728c6a1ff8901fc`.

Reply:
You're right, my bad. I'm not sure where I got those hashes from, I must have mistakenly copied the latest commit to `master` instead. I pushed the correct ones.